scripts/test_env.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+"""
+Setup or tear down a bare metal
+environment running a debug build of Iroha.
+"""
+
+import argparse
+import ipaddress
+import json
+import logging
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+
+
+class Network:
+    """
+    A network of bootstrapped peers to run on bare metal.
+    """
+    def __init__(self, setup_dir: pathlib.Path, host_addr: ipaddress.IPv4Address, n_peers: int = 4):
+        logging.info("Setting up test environment...")
+        self.setup_dir = setup_dir
+        self.n_peers = n_peers
+        peers_dir = setup_dir.joinpath("peers")
+        os.mkdir(peers_dir)
+        try:
+            shutil.copy2("./configs/peer/config.json", peers_dir)
+            shutil.copy2("./configs/peer/genesis.json", peers_dir)
+            shutil.copy2("./configs/peer/validator.wasm", peers_dir)
+        except FileNotFoundError:
+            logging.error("Some of the config files are missing. \
+                          Please provide them in the `./configs/peer` directory")
+            sys.exit(1)
+        try:
+            shutil.copy2("./target/debug/iroha", peers_dir)
+        except FileNotFoundError:
+            logging.error("Please build the `iroha` by running `cargo build --bin iroha`")
```

Comment:
Or you could prompt the user if you want to do that on their behalf. No harm in asking a `y/n` question.

Reply:
`click` library is great for that btw

Reply:
Was following the style of the original script here, but that makes sense. I'm trying to avoid using external dependencies though to make it runnable on bare systems.